interface.cpp

```diff
@@ -110,6 +121,74 @@ struct call_traits<'u'> {
         free(buf);
         return ret;
     }
+
+/* Python 3.5 introduced Py_EncodeLocale for converting wchar_t* to char*.
+   Backport that method for the benefit of Python 2.7 and 3.4. However,
+   if we're on Python 3.5+, just use the native method. */
+#if PY_MAJOR_VERSION == 2 || (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION <= 4)
+    static const char* to_bytes(std::wstring& value) {
```

Comment:
Is to_bytes just for raising the RuntimeError?

Reply:
Correct.